pkg/etcdcli/health.go

```diff
@@ -320,3 +320,7 @@ func (c *raftTermsCollector) Collect(ch chan<- prometheus.Metric) {
 		)
 	}
 }
+
+func MinimumTolerableQuorum(members int) int {
```

Comment:
can you please add a unit test for that?

Reply:
sure 👍🏽

Reply:
added @tjungblu 👍🏽